platform-api/che-core-api-builder/src/test/java/org/eclipse/che/api/builder/BuilderTest.java

```diff
@@ -95,6 +102,19 @@ public boolean removeListener(SourceManagerListener listener) {
                 }
             };
         }
+
+        @Override
+        public Map<String, BuilderEnvironment> getEnvironments() {
+            Map<String, BuilderEnvironment> env = new HashMap<>();
+
+            BuilderEnvironment builderEnvironment = DtoFactory.getInstance().createDto(BuilderEnvironment.class)
+                    .withId("default")
+                    .withIsDefault(true)
+                    .withDisplayName(this.getName());
+
+            env.put(builderEnvironment.getId(), builderEnvironment);
+            return env;
```

Comment:
as we build a map with only one entry : 
return Collections.singletonMap(key, value)  (no need to build HashMap, put key, value and return it)